external-plugins/coverage/plugin/handler/handler.go

```
@@ -22,6 +22,13 @@ import (
 	"sigs.k8s.io/prow/pkg/pjutil"
 )
 
+const (
+	defaultCoverageThreshold = 70
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)